packages/app/src/cli/services/info.ts

```diff
@@ -204,7 +204,9 @@ class AppInfo {
 
   accessScopesSection(): [string, string] {
     const title = 'Access Scopes in Root TOML File'
-    const lines = this.app.configuration.scopes.split(',').map((scope) => [scope])
+    const lines = getAppScopes(this.app.configuration)
```

Comment:
Can we add support and test coverage for '' in legacy TOML being interpreted as []